docs/contributor/pocs/integrate-prometheus-with-telemetry-manager-using-alerting.md

```diff
@@ -0,0 +1,258 @@
+# Integrate Prometheus with Telemetry Manager using Alerting
+
+## Goal
+
+The goal of the Proof of Concept is to test out integrating Prometheus into Telemetry Manager using Alerting.
+
+## Setup
+
+Follow these steps to set up the required environment:
+
+1. Create a Kubernetes cluster (k3d or Gardener).
+2. Create an overrides file specifically for the Prometheus Helm Chart. Save the file as `overrides.yaml`.
+```yaml
```

Comment:
for proper rendering below the numbered list item, the code sample needs 3 blank spaces at the beginning of each line